src/index.coffee

```diff
@@ -11,7 +11,7 @@ module.exports = class LiveScriptCompiler
 
   compile: (data, path, callback) ->
     try
-      result = LiveScript.compile data, bare: yes
+      result = livescript.compile data, bare: yes, header: no
```

Comment:
What if users want that? Perhaps it should be an option that gets exposed to the brunch config if this is something you want to use.